delivery_carrier_default_tracking_url/models/stock_picking.py

```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
+# Copyright 2019 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class StockPicking(models.Model):
+
+    _inherit = 'stock.picking'
+
+    default_tracking_url = fields.Char(
+        related='carrier_id.default_tracking_url',
+        readonly=True,
+        help="This is the default tracking url for this carrier."
```

Comment:
With the related, this help here is not necessary since it the same as in delivery.carrier (the related will copy it)